components/input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
+import { cn } from "../../lib/utils"
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ className, value, onChange, ...props }, ref) => {
+    return (
+      <input
+        value={value}
+        onChange={onChange}
```

Comment:
Playing whack-a-mole with these specific props! Like `type` and `placeholder`, I would also not destructure these and let `{...props}` take care of it.

Reply:
I would also move `{...props}` to the top right under `<input` so future developers can override them more easily if needed.